cloud/wfm/src/process/index.ts

```diff
@@ -0,0 +1,18 @@
+import {ProcessInstance} from '../process-instance';
+import {Task} from '../task';
+/**
+ * Definition holder for a linear set of {@link Task}s
+ * Intended to be instantiated as a {@link ProcessInstance} in order to be executed
+ */
+export interface Process {
+  /** Unique identifier for this {@link Process} */
+  id: string;
+  /** Description for UI */
+  displayName: string;
+  tasks: Task[];
```

Comment:
[Optional] Some implementation I saw store tasks in some additional container so different orders and flows can be applied later by providing separate implementation. 

Trick here is to not relay on the existence of the field and fact that is array. We can have simple TasksContainer that will just wrap array and provide methods to interact with it. 
That's just idea feel free to ignore.